ycbrcr.go

```diff
@@ -0,0 +1,143 @@
+// +build !go1.5
+
+package resize
+
+import "image"
+
+// YCbCr converts ycc to a YCbCr image with the same subsample ratio
+// as the YCbCr image that ycc was generated from.
+func (p *ycc) YCbCr() *image.YCbCr {
+	ycbcr := image.NewYCbCr(p.Rect, p.SubsampleRatio)
+	var off int
+
+	switch ycbcr.SubsampleRatio {
+	case image.YCbCrSubsampleRatio422:
+		for y := ycbcr.Rect.Min.Y; y < ycbcr.Rect.Max.Y; y++ {
```

Comment:
Thanks for the info. I'll probably have no cycles to look at it soon anyways. Hope to find some time next week to do it.